CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Rose
+
+## Prerequisites
+
+Make sure to have these tools installed:
+
+- [Git](https://git-scm.com/downloads)
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Node.js](https://nodejs.org/en/download) v16-v18
+  - [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
```

Comment:
This is implied by the link you give, but could you clarify that we specifically use Yarn v1.x, and also remove the `#mac-stable` from the link?